changelogs/unreleased/3829-unclear-error-when-compiler-venv-has-incompatible-python-version.yml

```diff
@@ -0,0 +1,7 @@
+description: An exception is now raised when there is a mismatch between the python versions of the compiler venv and the active process
```

Comment:
```suggestion
description: An exception is now raised when there is a mismatch between the python version of the compiler venv and the active process
```

Reply:
Sounds weird to me without the final "s" :thinking:

Reply:
A certain venv can only have one python version at a certain point in time, never multiple. As such, it doesn't make sense to use plural here, right?

Reply:
As I understand it there is one version for the compiler venv and one version for the active process, so 2 in total hence the plural.